dev/integration_tests/ios_add2app/ios_add2app.xcodeproj/project.pbxproj

```diff
@@ -213,7 +213,7 @@
 			buildConfigurationList = 24E221CB21A28A0C008ADF09 /* Build configuration list for PBXNativeTarget "ios_add2app" */;
 			buildPhases = (
 				4375EE880A727341E9C9A57D /* [CP] Check Pods Manifest.lock */,
-				CE6831C2A33C6C5C84521FDE /* [CP-User] Run Flutter Build Script */,
+				52B3E7E8995E582399C42407 /* [CP-User] Run Flutter Build ios_add2app_flutter Script */,
```

Comment:
This is an example host project, and was updated by running the integration tests.